pom.xml

```diff
@@ -53,6 +53,13 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>-Xmx512m</argLine>
```

Comment:
I tried a different technique (requires maven 3.3.1 or later). Didn't seem to resolve the issue for me.  

Refer to https://github.com/MarkEWaite/git-plugin/commit/8ad079cbbf636c98f311f3ce5437845d0523a0cb for the .mvn/jvm.config file technique that I used.  

The technique is described in the [maven 3.3.1 release notes](https://maven.apache.org/docs/3.3.1/release-notes.html#JVM_and_Command_Line_Options)